osi_object.proto

```diff
@@ -61,6 +61,22 @@ message StationaryObject
         //
         optional Color color = 4;
 
+        // The IDs of the lanes that the object is assigned to.
+        //
+        // \note Might be multiple if the object streches over multiple lanes.
```

Comment:
There's a typo.

```suggestion
        // \note Might be multiple if the object stretches over multiple lanes.
```

Reply:
The phrase _Might be multiple if..._ sounds strange to me, like something is missing (multiple _what_?). However, I see that this phrase has been used before elsewhere and I get what is meant.  If you want to change this, I recommend naming the thing there can be multiple of, for example, IDs.

```suggestion
        // \note Might be multiple IDs if the object stretches over multiple lanes.
```